packages/cli/src/util/dev/builder.ts

```diff
@@ -198,16 +198,12 @@ export async function executeBuild(
   }
 
   // Sort out build result to builder v2 shape
-  if (!builder.version || builder.version === 1) {
+  if (!builder.version || (builder as any).version === 1) {
```

Comment:
**question (non-blocking):** is `builder.version === 1` even possible? Should we augment the types?

Reply:
It's not strictly possible, but Builders are inherently "untrusted code" we should be defensive about it.